doc/source/schema/schema.md

```diff
@@ -36,33 +39,73 @@ The terminology used in this specification is based on [xarray's terminology](ht
 `InferenceData` stores all quantities that are relevant to fulfilling its goals in different groups. Different groups generally distinguish conceptually different quantities in Bayesian inference, however, convenience in {ref}`creation <creating_InferenceData>` and {ref}`usage <working_with_InferenceData>` of `InferenceData` objects also plays a role. In general, each quantity (such as posterior distribution or observed data) will be represented by several multidimensional labeled variables.
 
 ### Rules
-Following are a few rules which should be followed:
+Below are a few rules which should be followed:
 * Each group should have one entry per variable and each variable should be named.
-* When relevant, the first two dimensions of each variable should be the sample identifier (`chain`, `draw`).
-* For groups like `observed_data` or `constant_data`, the two initial dimensions(`chain`, `draw`) are omitted.
+* Dimension names `chain`, `draw`, `sample` and `post_pred_id` are reserved for
+  InferenceData use to indicate sample dimensions.
+* Dimensions in InferenceData (including sample dimensions) should be identified by name only.
+* For groups like `observed_data` or `constant_data`, all sample dimensions can be
+  omitted. For groups like `prior`, `posterior` or `posterior_predictive` either `sample` has to be
+  present or both `chain` and `draw` dimensions need to be present. Any combinations that follow
+  this are valid.
 * Dimensions must be named and share name with a coordinate specifying the index values, called coordinate values.
 * Coordinate values can be repeated and should not necessarily be numerical values.
 * Variables must not share names with dimensions.
-* Moreover, each group contains the following attributes:
-  - `created_at`: the date of creation of the group.
-  - `inference_library`: the library used to run the inference.
-  - `inference_library_version`: version of the inference library used.
+* Groups, variables or the InferenceData itself can have arbitrary metadata stored.
 
-### Relations
-`InferenceData` data objects contain any combination of the groups described below. There are also some relations (detailed below) between the variables and dimensions of different groups. Hence, whenever related groups are present they should comply with these relations.
+### Metadata
+No metadata is _required_ to be present in order to be compliant with the InferenceData schema.
+However, it is recommended to store the following fields when relevant:
+* `name`: InferenceData objects represent multiple quantities related to Bayesian modelling,
```

Comment:
Would `model_name` be more explicit that this is meant to be the name of the model, not the name of the group or dataset?

Reply:
Or is this metadata at the InferenceData level instead of at the group level?

Reply:
I was thinking about group level attributes here. I'll update the title. Maybe also add a note that the creation_library attrs might be stored in groups (they could be different between groups even though I don't expect this to be common)